tailwind.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   ],
   theme: {
     colors: {
+      transparent: 'transparent',
```

Comment:
Sneaking this one in here since it's useful

Reply:
It already exists as `eds-theme-color-transparent-black-0` (as `rgb(0,0,0,0)` but same same) but felt like using `transparent-black-0` for common usage felt unwieldy, can revert tho

Reply:
I wouldn't be opposed to having a token using the keyword as well? "transparent-black-0" is ... decidedly weird; its technically black but not practically. whereas `transparent` is just that. Much clearer AND exists like all our other tokens do, which i prefer

Reply:
@jinlee93 i think we can say the following: "tier 1 tokens are sometimes immutable and define fixed values."

We have a few like this already, like many of the z-index.json values, some of the typography values, etc. when the token name INCLUDES the value, it is immutable... maybe also a hint that its an anti-pattern but that's for a later discussion)

Reply:
@booc0mtaco I am going to remove this `transparent` line from this pr and add its a token in anther pr since it seems to be a need as a t(n) token in the system itself